dot/core/service.go

```diff
@@ -724,6 +748,7 @@ func (s *Service) ProcessTransactionMessage(msg network.Message) error {
 }
 
 // handle authority and randomness changes over transitions from one epoch to the next
+//nolint
```

Comment:
what are we ignoring?

Reply:
this function isn't used anywhere anymore, it was in `ProcessBlockResponseMessage` but it's been commented out since putting this isn't to digest isn't implemented yet so it just causes problems